### PR TITLE
Fix cats-as-a-service example code

### PR DESCRIPTION
**Pull request by patrick-codaio**
Fixes https://coda-final.oslash.com/o/bug/25759

(And I added some drive-by instructions to `make build` to ease fallout from https://github.com/coda/packs-sdk/pull/2204)

### CODE REVIEW

File: runtime/isolated-vm/package.json [hunk 1]
```diff
@@ -54,5 +54,5 @@
     "test": "node test.js || nodejs test.js"
   },
   "types": "isolated-vm.d.ts",
-  "version": "4.4.1"
+  "version": "4.4.2"
```
Comment (patrick-codaio) [inline runtime/isolated-vm/package.json:57]: I didn't exactly intend to update this, it's just what happened when I ran `yarn add isolated-vm`

**Review by huayang-codaio — APPROVED** [review]

**Review by ekoleda-codaio — APPROVED** [review]
Thanks for the fix @patrick-codaio !

File: runtime/isolated-vm/package.json (hunk 1 above)
Comment (huayang-codaio) [reply runtime/isolated-vm/package.json:57]: can you make bs locally? 

this PR doesn't change yarn.lock and isolated-vm is still 4.4.1 in yarn.lock

```
isolated-vm@^4.4.1:
  version "4.4.1"
  resolved "https://registry.yarnpkg.com/isolated-vm/-/isolated-vm-4.4.1.tgz#651ce31e435e769b2a356bb03db2553a478acebe"
  integrity sha512-5aDwxQGm78vHS+qJeUli2ILroG7OS/k3D/Mc0kcT9vyujiL4bV7PYYix1mAvuBm3v44nz2qcfAOqgAbhuACc/w==
```

Comment (patrick-codaio) [reply runtime/isolated-vm/package.json]: `make bs` had been succeeding locally, FWIW, but reverting the isolated-vm changes worked, thank you!

**Pull request merged**